### PR TITLE
Pattern/existing eks awsnative - WIP

### PR DESCRIPTION
**Pull request by nrajb**
*Issue #, if available:*
Close: #8 

*Description of changes:*
Added new pattern for `existing-eks` with `awsnative` observability tools

Test: 
Tested new pattern by deploying single cluster with `make pattern single-new-eks-cluster deploy` first

```
make pattern existing-eks-awsnative-observability-construct deploy


make pattern existing-eks-awsnative-observability-construct deploy                                                                                                                  [22/07/23|12:08pm]
existing-eks-awsnative-observability-construct performing deploy
node node_modules/.bin/cdk --app "npx ts-node bin/existing-eks-awsnative-observability-construct.ts"    deploy
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!                                                                                                                      !!
!!  This software has not been tested with node v20.4.0.                                                                !!
!!  Should you encounter odd runtime issues, please try using one of the supported release before filing a bug report.  !!
!!                                                                                                                      !!
!!  This software is currently running on node v20.4.0.                                                                 !!
!!  As of the current release of this software, supported node releases are:                                            !!
!!  - ^18.0.0 (Planned end-of-life: 2025-04-30)                                                                         !!
!!  - ^16.3.0 (Planned end-of-life: 2023-09-11)                                                                         !!
!!                                                                                                                      !!
!!  This warning can be silenced by setting the JSII_SILENCE_WARNING_UNTESTED_NODE_VERSION environment variable.        !!
!!                                                                                                                      !!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
INFO === Run make compile before each run, if any code modification was made. === 


looking up non-default vpc-02b1966207349917d VPC
2023-07-22 11:14:10.699 DEBUG           main    Adding dependency from CertManagerAddOn to AwsLoadBalancerControllerAddOn
2023-07-22 11:14:10.699 DEBUG           main    Adding dependency from CloudWatchLogsAddon to AwsLoadBalancerControllerAddOn
2023-07-22 11:14:10.699 DEBUG           main    Adding dependency from ContainerInsightsAddOn to AwsLoadBalancerControllerAddOn

✨  Synthesis time: 4.99s

existing-eks-awsnative-observability-accelerator:  start: Building 79d02d447d8d1f1f18e21aa2d2fccc3a55363ba4ac9815e598ebdaecee7f0577:***************-us-west-2
existing-eks-awsnative-observability-accelerator:  success: Built 79d02d447d8d1f1f18e21aa2d2fccc3a55363ba4ac9815e598ebdaecee7f0577:***************--us-west-2
existing-eks-awsnative-observability-accelerator:  start: Publishing 79d02d447d8d1f1f18e21aa2d2fccc3a55363ba4ac9815e598ebdaecee7f0577:***************--us-west-2
existing-eks-awsnative-observability-accelerator:  success: Published 79d02d447d8d1f1f18e21aa2d2fccc3a55363ba4ac9815e598ebdaecee7f0577:***************--us-west-2
This deployment will make potentially sensitive changes according to your current security approval level (--require-approval broadening).
Please confirm you intend to make the following modifications:

IAM Statement Changes
┌───┬─────────────────────────────────────────────┬────────┬─────────────────────────────────────────────┬─────────────────────────────────────────────┬────────────────────────────────────────────────┐
│   │ Resource                                    │ Effect │ Action                                      │ Principal                                   │ Condition                                      │
├───┼─────────────────────────────────────────────┼────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────────────────────┤
│ + │ ${AWSCDKCfnUtilsProviderCustomResourceProvi │ Allow  │ sts:AssumeRole                              │ Service:lambda.amazonaws.com                │                                                │
│   │ der/Role.Arn}                               │        │                                             │                                             │                                                │
├───┼─────────────────────────────────────────────┼────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────────────────────┤
│ + │ ${existing-eks-awsnative-observability-acce │ Allow  │ kms:*                                       │ AWS:arn:${AWS::Partition}:iam::*************** │                                                │
│   │ lerator-kms-key.Arn}                        │        │                                             │ :root                                       │                                                │
├───┼─────────────────────────────────────────────┼────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────────────────────┤
│ + │ ${imported-cluster-single-new-eks-observabi │ Allow  │ sts:AssumeRoleWithWebIdentity               │ Federated:arn:aws:iam::***************:oidc-pr │ "StringEquals": "${imported-cluster-single-new │
│   │ lity-accelerator/adot-collector-sa/Role.Arn │        │                                             │ ovider/oidc.eks.us-west-2.amazonaws.com/id/ │ -eks-observability-accelerator/adot-collector- │
│   │ }                                           │        │                                             │ ED68BF6140FDD5FF5F33F855F98FCC00            │ sa/ConditionJson/Resource.Value}"              │
├───┼─────────────────────────────────────────────┼────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────────────────────┤
│ + │ ${imported-cluster-single-new-eks-observabi │ Allow  │ sts:AssumeRoleWithWebIdentity               │ Federated:arn:aws:iam::***************:oidc-pr │ "StringEquals": "${imported-cluster-single-new │
│   │ lity-accelerator/aws-fluent-bit-for-cw-sa/R │        │                                             │ ovider/oidc.eks.us-west-2.amazonaws.com/id/ │ -eks-observability-accelerator/aws-fluent-bit- │
│   │ ole.Arn}                                    │        │                                             │ ED68BF6140FDD5FF5F33F855F98FCC00            │ for-cw-sa/ConditionJson/Resource.Value}"       │
├───┼─────────────────────────────────────────────┼────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────────────────────┤
│ + │ ${imported-cluster-single-new-eks-observabi │ Allow  │ sts:AssumeRoleWithWebIdentity               │ Federated:arn:aws:iam::***************:oidc-pr │ "StringEquals": "${imported-cluster-single-new │
│   │ lity-accelerator/aws-load-balancer-controll │        │                                             │ ovider/oidc.eks.us-west-2.amazonaws.com/id/ │ -eks-observability-accelerator/aws-load-balanc │
│   │ er/Role.Arn}                                │        │                                             │ ED68BF6140FDD5FF5F33F855F98FCC00            │ er-controller/ConditionJson/Resource.Value}"   │
├───┼─────────────────────────────────────────────┼────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────────────────────┤
│ + │ *                                           │ Allow  │ iam:CreateServiceLinkedRole                 │ AWS:${imported-cluster-single-new-eks-obser │ "StringEquals": {                              │
│   │                                             │        │                                             │ vability-accelerator/aws-load-balancer-cont │   "iam:AWSServiceName": "elasticloadbalancing. │
│   │                                             │        │                                             │ roller/Role}                                │ amazonaws.com"                                 │
│   │                                             │        │                                             │                                             │ }                                              │
│ + │ *                                           │ Allow  │ ec2:DescribeAccountAttributes               │ AWS:${imported-cluster-single-new-eks-obser │                                                │
│   │                                             │        │ ec2:DescribeAddresses                       │ vability-accelerator/aws-load-balancer-cont │                                                │
│   │                                             │        │ ec2:DescribeAvailabilityZones               │ roller/Role}                                │                                                │
│   │                                             │        │ ec2:DescribeCoipPools                       │                                             │                                                │
│   │                                             │        │ ec2:DescribeInstances                       │                                             │                                                │
│   │                                             │        │ ec2:DescribeInternetGateways                │                                             │                                                │
│   │                                             │        │ ec2:DescribeNetworkInterfaces               │                                             │                                                │
│   │                                             │        │ ec2:DescribeSecurityGroups                  │                                             │                                                │
│   │                                             │        │ ec2:DescribeSubnets                         │                                             │                                                │
│   │                                             │        │ ec2:DescribeTags                            │                                             │                                                │
│   │                                             │        │ ec2:DescribeVpcPeeringConnections           │                                             │                                                │
│   │                                             │        │ ec2:DescribeVpcs                            │                                             │                                                │
│   │                                             │        │ ec2:GetCoipPoolUsage                        │                                             │                                                │
│   │                                             │        │ elasticloadbalancing:DescribeListenerCertif │                                             │                                                │
│   │                                             │        │ icates                                      │                                             │                                                │
│   │                                             │        │ elasticloadbalancing:DescribeListeners      │                                             │                                                │
│   │                                             │        │ elasticloadbalancing:DescribeLoadBalancerAt │                                             │                                                │
│   │                                             │        │ tributes                                    │                                             │                                                │
│   │                                             │        │ elasticloadbalancing:DescribeLoadBalancers  │                                             │                                                │
│   │                                             │        │ elasticloadbalancing:DescribeRules          │                                             │                                                │
│   │                                             │        │ elasticloadbalancing:DescribeSSLPolicies    │                                             │                                                │
│   │                                             │        │ elasticloadbalancing:DescribeTags           │                                             │                                                │
│   │                                             │        │ elasticloadbalancing:DescribeTargetGroupAtt │                                             │                                                │
│   │                                             │        │ ributes                                     │                                             │                                                │
│   │                                             │        │ elasticloadbalancing:DescribeTargetGroups   │                                             │                                                │
│   │                                             │        │ elasticloadbalancing:DescribeTargetHealth   │                                             │                                                │
│ + │ *                                           │ Allow  │ acm:DescribeCertificate                     │ AWS:${imported-cluster-single-new-eks-obser │                                                │
│   │                                             │        │ acm:ListCertificates                        │ vability-accelerator/aws-load-balancer-cont │                                                │
│   │                                             │        │ cognito-idp:DescribeUserPoolClient          │ roller/Role}                                │                                                │
│   │                                             │        │ iam:GetServerCertificate                    │                                             │                                                │
│   │                                             │        │ iam:ListServerCertificates                  │                                             │                                                │
│   │                                             │        │ shield:CreateProtection                     │                                             │                                                │
│   │                                             │        │ shield:DeleteProtection                     │                                             │                                                │
│   │                                             │        │ shield:DescribeProtection                   │                                             │                                                │
│   │                                             │        │ shield:GetSubscriptionState                 │                                             │                                                │
│   │                                             │        │ waf-regional:AssociateWebACL                │                                             │                                                │
│   │                                             │        │ waf-regional:DisassociateWebACL             │                                             │                                                │
│   │                                             │        │ waf-regional:GetWebACL                      │                                             │                                                │
│   │                                             │        │ waf-regional:GetWebACLForResource           │                                             │                                                │
│   │                                             │        │ wafv2:AssociateWebACL                       │                                             │                                                │
│   │                                             │        │ wafv2:DisassociateWebACL                    │                                             │                                                │
│   │                                             │        │ wafv2:GetWebACL                             │                                             │                                                │
│   │                                             │        │ wafv2:GetWebACLForResource                  │                                             │                                                │
│ + │ *                                           │ Allow  │ ec2:AuthorizeSecurityGroupIngress           │ AWS:${imported-cluster-single-new-eks-obser │                                                │
│   │                                             │        │ ec2:RevokeSecurityGroupIngress              │ vability-accelerator/aws-load-balancer-cont │                                                │
│   │                                             │        │                                             │ roller/Role}                                │                                                │
│ + │ *                                           │ Allow  │ ec2:CreateSecurityGroup                     │ AWS:${imported-cluster-single-new-eks-obser │                                                │
│   │                                             │        │                                             │ vability-accelerator/aws-load-balancer-cont │                                                │
│   │                                             │        │                                             │ roller/Role}                                │                                                │
│ + │ *                                           │ Allow  │ ec2:AuthorizeSecurityGroupIngress           │ AWS:${imported-cluster-single-new-eks-obser │ "Null": {                                      │
│   │                                             │        │ ec2:DeleteSecurityGroup                     │ vability-accelerator/aws-load-balancer-cont │   "aws:ResourceTag/elbv2.k8s.aws/cluster": "fa │
│   │                                             │        │ ec2:RevokeSecurityGroupIngress              │ roller/Role}                                │ lse"                                           │
│   │                                             │        │                                             │                                             │ }                                              │
│ + │ *                                           │ Allow  │ elasticloadbalancing:CreateLoadBalancer     │ AWS:${imported-cluster-single-new-eks-obser │ "Null": {                                      │
│   │                                             │        │ elasticloadbalancing:CreateTargetGroup      │ vability-accelerator/aws-load-balancer-cont │   "aws:RequestTag/elbv2.k8s.aws/cluster": "fal │
│   │                                             │        │                                             │ roller/Role}                                │ se"                                            │
│   │                                             │        │                                             │                                             │ }                                              │
│ + │ *                                           │ Allow  │ elasticloadbalancing:CreateListener         │ AWS:${imported-cluster-single-new-eks-obser │                                                │
│   │                                             │        │ elasticloadbalancing:CreateRule             │ vability-accelerator/aws-load-balancer-cont │                                                │
│   │                                             │        │ elasticloadbalancing:DeleteListener         │ roller/Role}                                │                                                │
│   │                                             │        │ elasticloadbalancing:DeleteRule             │                                             │                                                │
│ + │ *                                           │ Allow  │ elasticloadbalancing:DeleteLoadBalancer     │ AWS:${imported-cluster-single-new-eks-obser │ "Null": {                                      │
│   │                                             │        │ elasticloadbalancing:DeleteTargetGroup      │ vability-accelerator/aws-load-balancer-cont │   "aws:ResourceTag/elbv2.k8s.aws/cluster": "fa │
│   │                                             │        │ elasticloadbalancing:ModifyLoadBalancerAttr │ roller/Role}                                │ lse"                                           │
│   │                                             │        │ ibutes                                      │                                             │ }                                              │
│   │                                             │        │ elasticloadbalancing:ModifyTargetGroup      │                                             │                                                │
│   │                                             │        │ elasticloadbalancing:ModifyTargetGroupAttri │                                             │                                                │
│   │                                             │        │ butes                                       │                                             │                                                │
│   │                                             │        │ elasticloadbalancing:SetIpAddressType       │                                             │                                                │
│   │                                             │        │ elasticloadbalancing:SetSecurityGroups      │                                             │                                                │
│   │                                             │        │ elasticloadbalancing:SetSubnets             │                                             │                                                │
│ + │ *                                           │ Allow  │ elasticloadbalancing:AddListenerCertificate │ AWS:${imported-cluster-single-new-eks-obser │                                                │
│   │                                             │        │ s                                           │ vability-accelerator/aws-load-balancer-cont │                                                │
│   │                                             │        │ elasticloadbalancing:ModifyListener         │ roller/Role}                                │                                                │
│   │                                             │        │ elasticloadbalancing:ModifyRule             │                                             │                                                │
│   │                                             │        │ elasticloadbalancing:RemoveListenerCertific │                                             │                                                │
│   │                                             │        │ ates                                        │                                             │                                                │
│   │                                             │        │ elasticloadbalancing:SetWebAcl              │                                             │                                                │
│ + │ *                                           │ Allow  │ ec2:DescribeTags                            │ AWS:${imported-cluster-single-new-eks-obser │                                                │
│   │                                             │        │ ec2:DescribeVolumes                         │ vability-accelerator/aws-fluent-bit-for-cw- │                                                │
│   │                                             │        │ logs:CreateLogGroup                         │ sa/Role}                                    │                                                │
│   │                                             │        │ logs:CreateLogStream                        │                                             │                                                │
│   │                                             │        │ logs:DescribeLogGroups                      │                                             │                                                │
│   │                                             │        │ logs:DescribeLogStreams                     │                                             │                                                │
│   │                                             │        │ logs:PutLogEvents                           │                                             │                                                │
├───┼─────────────────────────────────────────────┼────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────────────────────┤
│ + │ arn:${AWS::Partition}:ec2:*:*:security-grou │ Allow  │ ec2:CreateTags                              │ AWS:${imported-cluster-single-new-eks-obser │ "StringEquals": {                              │
│   │ p/*                                         │        │                                             │ vability-accelerator/aws-load-balancer-cont │   "ec2:CreateAction": "CreateSecurityGroup"    │
│   │                                             │        │                                             │ roller/Role}                                │ },                                             │
│   │                                             │        │                                             │                                             │ "Null": {                                      │
│   │                                             │        │                                             │                                             │   "aws:RequestTag/elbv2.k8s.aws/cluster": "fal │
│   │                                             │        │                                             │                                             │ se"                                            │
│   │                                             │        │                                             │                                             │ }                                              │
│ + │ arn:${AWS::Partition}:ec2:*:*:security-grou │ Allow  │ ec2:CreateTags                              │ AWS:${imported-cluster-single-new-eks-obser │ "Null": {                                      │
│   │ p/*                                         │        │ ec2:DeleteTags                              │ vability-accelerator/aws-load-balancer-cont │   "aws:RequestTag/elbv2.k8s.aws/cluster": "tru │
│   │                                             │        │                                             │ roller/Role}                                │ e",                                            │
│   │                                             │        │                                             │                                             │   "aws:ResourceTag/elbv2.k8s.aws/cluster": "fa │
│   │                                             │        │                                             │                                             │ lse"                                           │
│   │                                             │        │                                             │                                             │ }                                              │
├───┼─────────────────────────────────────────────┼────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────────────────────┤
│ + │ arn:${AWS::Partition}:elasticloadbalancing: │ Allow  │ elasticloadbalancing:AddTags                │ AWS:${imported-cluster-single-new-eks-obser │                                                │
│   │ *:*:listener-rule/app/*/*/*                 │        │ elasticloadbalancing:RemoveTags             │ vability-accelerator/aws-load-balancer-cont │                                                │
│   │ arn:${AWS::Partition}:elasticloadbalancing: │        │                                             │ roller/Role}                                │                                                │
│   │ *:*:listener-rule/net/*/*/*                 │        │                                             │                                             │                                                │
│   │ arn:${AWS::Partition}:elasticloadbalancing: │        │                                             │                                             │                                                │
│   │ *:*:listener/app/*/*/*                      │        │                                             │                                             │                                                │
│   │ arn:${AWS::Partition}:elasticloadbalancing: │        │                                             │                                             │                                                │
│   │ *:*:listener/net/*/*/*                      │        │                                             │                                             │                                                │
├───┼─────────────────────────────────────────────┼────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────────────────────┤
│ + │ arn:${AWS::Partition}:elasticloadbalancing: │ Allow  │ elasticloadbalancing:AddTags                │ AWS:${imported-cluster-single-new-eks-obser │ "Null": {                                      │
│   │ *:*:loadbalancer/app/*/*                    │        │ elasticloadbalancing:RemoveTags             │ vability-accelerator/aws-load-balancer-cont │   "aws:RequestTag/elbv2.k8s.aws/cluster": "tru │
│   │ arn:${AWS::Partition}:elasticloadbalancing: │        │                                             │ roller/Role}                                │ e",                                            │
│   │ *:*:loadbalancer/net/*/*                    │        │                                             │                                             │   "aws:ResourceTag/elbv2.k8s.aws/cluster": "fa │
│   │ arn:${AWS::Partition}:elasticloadbalancing: │        │                                             │                                             │ lse"                                           │
│   │ *:*:targetgroup/*/*                         │        │                                             │                                             │ }                                              │
│ + │ arn:${AWS::Partition}:elasticloadbalancing: │ Allow  │ elasticloadbalancing:AddTags                │ AWS:${imported-cluster-single-new-eks-obser │ "StringEquals": {                              │
│   │ *:*:loadbalancer/app/*/*                    │        │                                             │ vability-accelerator/aws-load-balancer-cont │   "elasticloadbalancing:CreateAction": [       │
│   │ arn:${AWS::Partition}:elasticloadbalancing: │        │                                             │ roller/Role}                                │     "CreateTargetGroup",                       │
│   │ *:*:loadbalancer/net/*/*                    │        │                                             │                                             │     "CreateLoadBalancer"                       │
│   │ arn:${AWS::Partition}:elasticloadbalancing: │        │                                             │                                             │   ]                                            │
│   │ *:*:targetgroup/*/*                         │        │                                             │                                             │ },                                             │
│   │                                             │        │                                             │                                             │ "Null": {                                      │
│   │                                             │        │                                             │                                             │   "aws:RequestTag/elbv2.k8s.aws/cluster": "fal │
│   │                                             │        │                                             │                                             │ se"                                            │
│   │                                             │        │                                             │                                             │ }                                              │
├───┼─────────────────────────────────────────────┼────────┼─────────────────────────────────────────────┼─────────────────────────────────────────────┼────────────────────────────────────────────────┤
│ + │ arn:${AWS::Partition}:elasticloadbalancing: │ Allow  │ elasticloadbalancing:DeregisterTargets      │ AWS:${imported-cluster-single-new-eks-obser │                                                │
│   │ *:*:targetgroup/*/*                         │        │ elasticloadbalancing:RegisterTargets        │ vability-accelerator/aws-load-balancer-cont │                                                │
│   │                                             │        │                                             │ roller/Role}                                │                                                │
└───┴─────────────────────────────────────────────┴────────┴─────────────────────────────────────────────┴─────────────────────────────────────────────┴────────────────────────────────────────────────┘
IAM Policy Changes
┌───┬─────────────────────────────────────────────────────────────────────────────────────┬──────────────────────────────────────────────────────────────────────────────────────────────┐
│   │ Resource                                                                            │ Managed Policy ARN                                                                           │
├───┼─────────────────────────────────────────────────────────────────────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────┤
│ + │ ${AWSCDKCfnUtilsProviderCustomResourceProvider/Role}                                │ {"Fn::Sub":"arn:${AWS::Partition}:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole"} │
├───┼─────────────────────────────────────────────────────────────────────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────┤
│ + │ ${imported-cluster-single-new-eks-observability-accelerator/adot-collector-sa/Role} │ arn:${AWS::Partition}:iam::aws:policy/CloudWatchAgentServerPolicy                            │
└───┴─────────────────────────────────────────────────────────────────────────────────────┴──────────────────────────────────────────────────────────────────────────────────────────────┘
(NOTE: There may be security-related changes not in this list. See https://github.com/aws/aws-cdk/issues/1299)

Do you wish to deploy these changes (y/n)? y
existing-eks-awsnative-observability-accelerator: deploying... [1/1]
existing-eks-awsnative-observability-accelerator: creating CloudFormation changeset...

 ✅  existing-eks-awsnative-observability-accelerator

✨  Deployment time: 153.1s

Stack ARN:
arn:aws:cloudformation:us-west-2:***************:stack/existing-eks-awsnative-observability-accelerator/eb132750-2880-11ee-aad8-0283ac5cebb3

✨  Total time: 158.09s

```
Screenshots:
image

image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: lib/existing-eks-awsnative-observability-construct/index.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            clusterSecurityGroupId: sdkCluster.resourcesVpcConfig?.clusterSecurityGroupId
+        });
+
+        const addOns: Array<blueprints.ClusterAddOn> = [
+            new blueprints.addons.CloudWatchLogsAddon({
+                logGroupPrefix: `/aws/eks/${stackId}`,
+                logRetentionDays: 30
+            }),
+            new blueprints.addons.ContainerInsightsAddOn(),
+            //new blueprints.addons.XrayAddOn(),
```
Comment (shapirov103) [inline lib/existing-eks-awsnative-observability-construct/index.ts:42]: Let's remove the commented code.

**Review by shapirov103 — COMMENTED** [review]
Let's add a paragraph to the doc about XRay addon being atm incompatible with imported clusters.

**Comment by nrajb** [comment]
> @nrajb I think XRAY Addon should work with Existing cluster too. Did you get a chance to test it? I agree to Mikhail's suggestion. We should add a para describing why if it wont work. Also We should create an issue if this is something coming soon.

@elamaran11 yes, I tried to deploy with Xray Addon enabled:
        ` new blueprints.addons.XrayAddOn(),`

but got below error:
```
2023-07-22 12:17:39.987 INFO            main    [
  'Existing Cluster Pattern is missing information of existing cluster: Error: XrayAddOn is supported with EKS EC2 only and is not supported for imported clusters'
]
```

I will update the PR with the doc as per suggestions soon!

**Comment by elamaran11** [comment]
@nrajb Could we close this current week?

**Comment by nrajb** [comment]
Created new [PR](https://github.com/aws-observability/cdk-aws-observability-accelerator/pull/80), closing this one

**Pull request closed without merging**